request_handlers_test.cc

```diff
@@ -23,77 +24,84 @@ class RequestHandlersTest : public ::testing::Test {
 };
 
 TEST_F(RequestHandlersTest, EmptyStringTest) {
-  char request_buffer[MAX_REQ_SIZE];
+  char request_buffer[] = "";
+  const size_t request_buffer_size = 0;
 
-  // Customize message in request buffer
-  strcpy(request_buffer, "");
+  // The request_buffer is normally filled by a socket-read
+  // The response_buffer is allocated by the request-handler
+  char* response_buffer = nullptr;
+  size_t response_buffer_size = 0;
 
-  // Handle echo response in external handler
-  char response_buffer[MAX_REQ_SIZE];
-  size_t response_size = 0;
+  char expected_response_buffer[] =
```

Comment:
Can we make this a `const char[]`? Expected data normally doesn't change over the course of a test.